model_compression_toolkit/common/quantization/quantization_params_generation/kl_selection.py

```diff
@@ -385,3 +395,36 @@ def _get_sliced_histogram(bins: np.ndarray,
     counts_subset = deepcopy(counts[first_bin_idx:last_bin_idx])
 
     return bins_subset, counts_subset
+
+
+def _kl_batch_error_function(x: np.ndarray,
+                             q_x: np.ndarray,  # dummy
+                             range_min: np.ndarray,
+                             range_max: np.ndarray,
+                             n_bins: int = 2048,
+                             n_bits: int = 8) -> np.float:
+    """
+    Vectorized implementation for the KL error function,
+    to allow vectorized per-channel computation (for weights quantization).
+    Input tensor x is expected to be reshaped before passed to this method,
+    such that axis 0 contains the number of channels and the tensor values are placed along axis 1.
+    Note that the quantized value of x is not directly used in this method,
+    but rather being calculated during the process of gathering the distribution statistics
+    within the _kl_error_function.
+
+    Args:
+        x:
```

Comment:
Args comments are missing:
x is a ndarray of shape n x m where n is the number of channels and m is the number of elements in each channel